pyproject.toml

```diff
@@ -15,6 +15,8 @@ yaspin = "^1.0.0"
 redis = {version="*", optional=true}
 pika = {version="*", optional=true}
 kafka-python = {version="*", optional=true}
+azure-storage-blob = { version = "^12.3.1", optional = true }
+cffi = {version = "^1.14.0", optional = true}
```

Comment:
should be * to support compatibility

Reply:
Dunno, internal stuff might not work in case of breakage.